<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите ID изменяемой записи</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="id_edit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>431</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="accept_button">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>271</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Принять изменения</string>
    </property>
   </widget>
   <widget class="QPushButton" name="show_button">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Отобразить</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>671</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Внесите данные для записи в базу данных</string>
    </property>
   </widget>
   <widget class="QPushButton" name="make_button">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>360</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить запись в базу</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="edit_table">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>741</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="insert_table">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>751</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
